examples/spe1/resources/SPE1CASE2.DATA.jinja2

```diff
@@ -0,0 +1,436 @@
+-- This reservoir simulation deck is made available under the Open Database
```

Comment:
Jinja template based on `SPE1CASE2` (https://github.com/OPM/opm-data/blob/master/spe1/SPE1CASE2.DATA). Currently the porosity of the field is parameterised.